src/app/util/helper-functions.ts

```diff
@@ -73,4 +73,9 @@ export function parseLogData(logs: string[], isType: InformationSourceType = nul
             wrapper.substr(wrapper.indexOf(' ') + 1)
         );
     });
+}
+
+export function getUserAvatarUri(user) {
+    const avatarSelector = user != undefined ? (user.firstName[0].charCodeAt(0) + user.lastName[0].charCodeAt(0)) % 5 : 0;
+    return "assets/avatars/" + avatarSelector + ".png";
```

Comment:
with this approach, only an undefined user will result in the default 0.
Might run into error for a user not present in the sqlliste db (kratos) but in the db (imported users)

```suggestion
    let avatarId = 0;
    if (user && user.firstName && user.lastName){
        avatarId = user.firstName[0].charCodeAt(0) + user.lastName[0].charCodeAt(0)) % 5;
    }
    return "assets/avatars/" + avatarSelector + ".png";
```